src/PoCs/failed.tsx

```diff
@@ -0,0 +1,15 @@
+import { Split, SplitItem, Label } from "@patternfly/react-core";
+import React, {VoidFunctionComponent } from "react";
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import "./style.css";
+
+
+export const InstanceCreationFailed: VoidFunctionComponent  = () => {
+    return (
+        <div>       
+            <Label color="red" icon={<ExclamationCircleIcon />}>
+                Failed
+            </Label>    
+        </div>
```

Comment:
We shouldn't use the `<Label>` component for this status. Instead, the same  structure that you use for the Ready status can be used here.